### PR TITLE
lokalise2: init at 2.6.0

### PR DESCRIPTION
**Pull request by timstott**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).
- [x] Add license (blocked on lokalise/lokalise-cli-2-go#38)

### CODE REVIEW

File: pkgs/tools/misc/lokalise2-cli/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  vendorSha256 = "09qkh9liba4g8qakzgsszzf9lsg1k8x6biny868d390crp8dn1ac";
+
+  postInstall = ''
+    mv $out/bin/lokalise-cli-2-go $out/bin/lokalise2
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://lokalise.com";
+    description = "Translation platform for developers. Upload language files, translate, integrate via API.";
```
Comment (timstott) [inline pkgs/tools/misc/lokalise2-cli/default.nix:24]: The folks at Lokalise have yet to provide a license lokalise/lokalise-cli-2-go#38 :shrug:

File: pkgs/tools/misc/lokalise2-cli/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "lokalise2-cli";
+  version = "2.6.0";
+
+  goPackagePath = "github.com/lokalise/lokalise-cli-2-go";
```
Comment (jonringer) [inline pkgs/tools/misc/lokalise2-cli/default.nix:7]: this doesn't get used with `buildGoModule`

Comment (timstott) [reply pkgs/tools/misc/lokalise2-cli/default.nix]: Thank you for your review @jonringer. I've amended the commit your feedback.

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 93546 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>1 package built:</summary><br>- lokalise2-cli</details>

**Pull request merged**